packages/oas-normalize/src/index.ts

```diff
@@ -19,6 +19,7 @@ import {
   stringToJSON,
 } from './lib/utils.js';
 
+// biome-ignore lint/style/noDefaultExport: This is fine for now.
```

Comment:
Every place I'm suppressing this would be a breaking change that I don't really feel like undertaking right now because upgrading `oas` a major version across our stack is a real pain and involves bumping `api` along the way.

Reply:
agreed w this assessment